student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
```

Comment:
- [ ] nice job. please kindly add setter and getter for @classroom
eg
`attr_reader :classroom`

Reply:
Hi @tufoinnkuo10 Thanks for the reminder, It's done. 😄